libblkid/src/superblocks/bcache.c

```diff
@@ -168,6 +168,17 @@ static void probe_bcachefs_sb_members(blkid_probe pr,
 	blkid_probe_set_fssize(pr, sectors * BCACHEFS_SECTOR_SIZE);
 }
 
+static int ptr_plus_offset_within_range(void *ptr, uint64_t offset, void *end)
```

Comment:
It would be better to use `is_` prefix for this kind of function (for example, is_within_range(start, size, end)).

Reply:
Indeed, done.